docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -17,86 +16,130 @@
    openvino_docs_MO_DG_prepare_model_Model_Optimizer_FAQ
 
 
-Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, 
-performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
+To convert a model to IR, you can use the following command:
```

Comment:
Not exactly to IR, but to OpenVINO model. IR is just a serialized representation of the model and it produced immediately by `mo`, but not by `mo.convert_model` which will be showed by default in the next example block.

Reply:
> Not exactly to IR, but to OpenVINO model. IR is just a serialized representation of the model and it produced immediately by `mo`, but not by `mo.convert_model` which will be showed by default in the next example block.

```suggestion
To convert a model to OpenVINO model format (``ov.Model``), you can use the following command:
```